tinymce-per-block/src/blocks/image-block/index.js

```diff
@@ -72,5 +72,13 @@ registerBlock( 'image', {
 			attrs: { /* caption: block.caption ,*/ align: block.align },
 			rawContent
 		};
+	},
+	create: () => {
+		return {
```

Comment:
mmm, nice idea I was hoping to be able to use only the `WPBlock -> WPBlock` initialization flow but this introduces some coupling between the source and the destination block.

---
Back to the `create` problem above. As you said, this is probably mergeable with `parse`. I have a personal preference over having it separate instead of handling an empty block as input. We could reconsider later.